mochad/L_Mochad1.lua

```diff
@@ -440,16 +441,17 @@ end
 function incoming(lul_data)
     local data = tostring(lul_data)
     log('Recieved '..data, 50)
-    local t = split_deliminated_string(data,',')
+    local t = split_deliminated_string(data,' ')
```

Comment:
I use a forked version of MOCHAD https://github.com/bfocht/mochad where I inserted commas which is why I parse on commas